examples/pybullet_industrial_intro.py

```diff
@@ -0,0 +1,167 @@
+
+b_inner_y = [1.22154196395355, 1.2242630523889242, 1.2269841408242983, 1.2324263176950467, 1.240589583001169, 1.2487528483072916, 1.2623582904841624, 1.2814059095317814, 1.2977324401440262, 1.3222222360623936, 1.3439909435453867, 1.3603174741576316, 1.3739229163345024, 1.3793650932052508, 1.382086181640625, 1.3875283585113731, 1.3875283585113731, 1.384807270075999, 1.382086181640625, 1.3712018278991283, 1.363038562593006, 1.3521542088515093, 1.3358276782392644, 1.3167800591916454, 1.2977324401440262, 1.2759637326610331, 1.256916113613414, 1.2433106714365434, 1.2324263176950467, 1.2242630523889242]
+b_inner_x= [0.46656534954407297, 0.4969604863221884, 0.5121580547112462, 0.5334346504559271, 0.5455927051671733, 0.560790273556231, 0.5729483282674772, 0.5820668693009119, 0.5881458966565349, 0.5851063829787234, 0.5759878419452887, 0.5638297872340425, 0.5455927051671733, 0.5303951367781155, 0.5182370820668692, 0.5, 0.4726443768996961, 0.4513677811550152, 0.4331306990881459, 0.40577507598784196, 0.3905775075987842, 0.378419452887538, 0.3662613981762918, 0.3601823708206687, 0.3601823708206687, 0.3662613981762918, 0.38145896656534956, 0.39969604863221886, 0.4209726443768997, 0.44224924012158057]
+
+k_y = [1.5752834605521895, 1.711337882320897, 1.7086167938855228, 1.8229025081712371, 1.9725623721168153, 1.8365079503481079, 1.9807256374229378, 1.8256235966066112, 1.711337882320897, 1.711337882320897, 1.5752834605521895]
+k_x = [0.8860182370820668, 0.8860182370820668, 0.5121580547112462, 0.6945288753799392, 0.6914893617021276, 0.4969604863221884, 0.2629179331306991, 0.25987841945288753, 0.4696048632218845, 0.25987841945288753, 0.25987841945288753]
+
+def build_path(x_data, y_data,number_of_steps):
+    z_data=[]
+    
+    i=0
+    path_list=[]
+    while i<len(x_data):
+        if i<len(x_data)-1:
+            sub_path=pi.linear_interpolation(np.array([2.5-x_data[i], y_data[i]-1, height]),np.array([2.5-x_data[i+1], y_data[i+1]-1, height]),number_of_steps)
+        if i==len(x_data)-1:
+            sub_path=pi.linear_interpolation(np.array([2.5-x_data[i], y_data[i]-1, height]),np.array([2.5-x_data[0], y_data[0]-1, height]),number_of_steps)
+        path_list.append(sub_path)
+        i+=1
+    path=path_list[0]
+    i=1
+    while i<len(path_list):
```

Comment:
while loops should be used with caution since they can run endlessly, consider using a for loop if possible.